docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md

```diff
@@ -0,0 +1,17 @@
+# Automatic_sp_privileges configuration for MySQL
+
+
+## Description
+
+The automatic_sp_privileges when enabled, causes automatic grants or revokes of the EXECUTE and ALTER ROUTINE privileges to the creator of a stored routine upon creation or removal of the routine. This check verifies that the automatic_sp_privileges is ON and alerts otherwise.
```

Comment:
Maybe add a link to
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges

The [automatic_sp_privileges](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges), when enabled, automatically grants `EXECUTE` and `ALTER_ROUTINE` privileges to the stored procedure creator. Also, when the routine is dropped, the privileges are dropped from the creator.

Reply:
@kedarvj ,  pls check this one too

Reply:
yeah... so I agree with the link addition for further reference.

Reply:
Updated the md